### PR TITLE
#549 - Added JoinedPermissions

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #549 
Added `JoinedPermissions` class to be used for composing global permissions and repo permissions

### CODE REVIEW

File: src/main/java/com/artipie/auth/JoinedPermissions.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+     * Origin permissions.
+     */
+    private final Collection<Permissions> origins;
+
+    /**
+     * Ctor.
+     *
+     * @param origins Origin permissions.
+     */
+    public JoinedPermissions(final Collection<Permissions> origins) {
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/auth/JoinedPermissions.java:47]: @olegmoz I'd suggest to add secondary ctor from `Permissions... origins`

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@olegmoz thanks, just one suggestion, please, take a look

**Comment by olegmoz** [comment]
@olenagerasimova thanks, added varargs constructor. Please review

**Review by olenagerasimova — APPROVED** [review]
@olegmoz thanks, good to merge

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#556` is not assigned, can't get performer

**Pull request merged**